ICVL.py

```diff
@@ -0,0 +1,206 @@
+import matplotlib.pyplot as plt
+import numpy as np
+from model_objects import model_parameters
+from DataSetInfoAbstractClass import dataSetInfoAbstract
+import seaborn as sns
+import cPickle
+
+
+class dataInfo(dataSetInfoAbstract):
+
+    def __init__(self):
+        self.name = "ICVL"
+
+    def load(self):
+
+        with open("Data/ICVL_Data/Training/ICVL_Training.pkl", "rb") as openfileobject:
+            (x_train, a_train) = cPickle.load(openfileobject)
+
+        with open("Data/ICVL_Data/Testing/ICVL_Testing.pkl", "rb") as openfileobject:
+            (x_test, a_test) = cPickle.load(openfileobject)
+        return (x_train, a_train, x_test, a_test)
+
+    # This file plots the lines for the various fingers. This is all hardcoded
+    # from the files
+    def draw_hands(self, Xs, Ys):
+        plt.scatter(Xs, Ys)
+
+        plt.gca().invert_yaxis()
+
+        linesStart = np.array([Xs[0], Xs[1]])
```

Comment:
You could probably shorten it by making several `for` loops and using arrays with the indices inside, but you wouldn't shorten it by very much (Because of specific colorings). If you were to implement a condition for checking which color to make it, I think you would end up with a somewhat complicated `for` loop. Because of the nature of the data's layout, there's not a particularly elegant way of addressing this, so far as I can see. I think that right now it's more readable (albeit ugly) than if you were to make a more complex, shorter way to do it.